packages/cashier/src/Stores/__tests__/crypto-fiat-converter-store.spec.js

```diff
@@ -0,0 +1,323 @@
+        WS.send.mockResolvedValue({
+            exchange_rates: {
+                base_currency: 'USD',
+                date: 1650868989,
+                rates: {
+                    AED: 3.67,
+                },
+            },
+        });
+
+        await crypto_fiat_converter_store.onChangeConverterToAmount({ target: { value: 3 } }, 'USD', 'AED');
+
+        expect(
+            crypto_fiat_converter_store.root_store.modules.cashier.general_store.percentageSelectorSelectionStatus
+        ).toBeCalledTimes(1);
+        expect(
+            crypto_fiat_converter_store.root_store.modules.cashier.general_store.calculatePercentage
+        ).toBeCalledTimes(1);
+        expect(crypto_fiat_converter_store.converter_to_amount).toEqual(3);
+    });
+
+    it('should set converter_from_error with error', async () => {
+        crypto_fiat_converter_store.setConverterToError(new Error('something went wrong'));
+
+        await crypto_fiat_converter_store.onChangeConverterToAmount({ target: { value: 3 } }, 'USD', 'AED');
+
+        expect(crypto_fiat_converter_store.converter_from_amount).toEqual('');
+        expect(crypto_fiat_converter_store.converter_from_amount).toEqual('');
+        expect(crypto_fiat_converter_store.converter_from_error).toEqual('');
+        expect(crypto_fiat_converter_store.is_timer_visible).toBeFalse();
+        expect(
+            crypto_fiat_converter_store.root_store.modules.cashier.account_transfer.setAccountTransferAmount
+        ).toBeCalledTimes(1);
+    });
+
+    it('should set converter_from_amount with an amount', async () => {
+        WS.send.mockResolvedValue({
+            exchange_rates: {
+                base_currency: 'USD',
+                date: 1650868989,
+                rates: {
+                    AED: 3.67,
+                },
+            },
+        });
+
+        await crypto_fiat_converter_store.onChangeConverterToAmount({ target: { value: 3 } }, 'USD', 'AED');
+
+        expect(
+            crypto_fiat_converter_store.root_store.modules.cashier.general_store.percentageSelectorSelectionStatus
+        ).toBeCalledTimes(1);
+        expect(
+            crypto_fiat_converter_store.root_store.modules.cashier.general_store.calculatePercentage
+        ).toBeCalledTimes(1);
+        expect(crypto_fiat_converter_store.converter_to_amount).toEqual(3);
+        expect(crypto_fiat_converter_store.converter_from_amount).toEqual('11.01');
+        expect(crypto_fiat_converter_store.converter_from_error).toEqual('');
+        expect(crypto_fiat_converter_store.is_timer_visible).toBeTrue();
+        expect(
+            crypto_fiat_converter_store.root_store.modules.cashier.account_transfer.setAccountTransferAmount
+        ).toBeCalledTimes(1);
+    });
+
+    it('should set is_timer_visible to false when converter_from_error has error', async () => {
+        crypto_fiat_converter_store.setConverterFromError(new Error('something went wrong'));
+        WS.send.mockResolvedValue({
```

Comment:
@vinu-deriv this ws.send response saw 6 times in your code, maybe it will be better to move this ws response into before each section?
beforeEach(() => {
    WS = {
        send: jest.fn.mockResolvedValue({
            exchange_rates: {
                base_currency: 'USD',
                date: 1650868989,
                rates: {
                    AED: 3.67,
                },
            },
        });
    };